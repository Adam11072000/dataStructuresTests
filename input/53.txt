init 23
getHighestScoredLabel 0 26
quit