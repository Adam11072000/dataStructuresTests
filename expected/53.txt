init done.
getHighestScoredLabel: INVALID_INPUT
quit done.
